package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@oclif/help",
+  "description": "standard help for oclif",
+  "version": "1.0.0",
+  "author": "Salesforce",
+  "bugs": "https://github.com/oclif/help/issues",
+  "dependencies": {
+    "@oclif/config": "1.18.2",
+    "@oclif/errors": "1.3.5",
+    "chalk": "^4.1.2",
+    "indent-string": "^4.0.0",
+    "lodash": "^4.17.21",
+    "string-width": "^4.2.0",
+    "strip-ansi": "^6.0.0",
+    "widest-line": "^3.1.0",
+    "wrap-ansi": "^6.2.0"
+  },
+  "devDependencies": {
+    "@oclif/command": "^1.8.13",
```

Comment:
This will still cause a circular dependency. Could we refactor the tests to not use it?

Reply:
This is part of the risk in our current dev practices. This form will not cause issues with customers during install, but may affect devs if installing for dev work on the module.